src/pages/Home/CustomizeBobaModalBody.tsx

```diff
@@ -3,18 +3,21 @@ import Form from 'react-bootstrap/Form';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import { Container } from 'react-bootstrap';
-import { milkList, toppingsList } from 'assets/sampleBobaAPI';
 import RadioCheckboxGroup from 'components/RadioCheckboxGroup';
 import './CustomizeBobaModalBody.css';
 
 interface BobaModalProps {
+  milkList: { [key: string]: { name: string } };
+  toppingsList: { [key: string]: { name: string; price: number } };
```

Comment:
why are you adding these?
you already have milk and toppings props for these.
It appears you just added to ignore TS errors. There's a reason behind why TS had errors, which cause the bug in your custom modal

Reply:
![image](https://user-images.githubusercontent.com/10414451/232358879-22217044-b32d-4346-a788-9fb3219b8cba.png)

Reply:
i am grabbing from home and passing it though pass to custom

Reply:
Aint you doing that before already? What have changed that require you doing this now?